05LessonFive/Checkpoint1/css/style.css

```diff
@@ -1,43 +1,31 @@
-.lawn {
+body {
     width: 100%;
-    height: 300px;
+    margin: auto;
 }
-
-.lawn h1 {
-    text-align: center;
-    font-family: monospace;
-    color: darkgray;
-}
-
-.lawn {
-    background-image: url(../img/banner-garden-stmichaelsmount.jpg);
-    background-repeat: no-repeat;
-    background-position: center;
-    background-size: cover;
+img {
+    width: 500px;
 }
-
-.left {
-    float: left;
-    width: 60%;
-    height: 300px;
-    margin-left: 260px;
-    background-color: lightgray;
+section {
+  padding-left: 5px;
+  padding-top: 5px;
 }
-
-.left h1 {
+td {
     text-align: center;
+    background-color: #F0F0F0;
+    float: auto;
```

Comment:
the `float` property does not accept `auto` as a value. remember that we review only 3 possible values